src/schema/schema.graphql

```diff
@@ -2453,6 +2454,12 @@ input InviteReplyInput {
 	exceptId: InstanceDate
 }
 
+input PurgetRevisionInput {
+	id: ID! 
+	ver: Int!
+	includeOlderRevisions: Int
```

Comment:
shouldn't this be BOOLEAN?

Reply:
SOAP api is accepting this flag value as '0|1' hence sending it as Int.
See this https://files.zimbra.com/docs/soap_api/9.0.0/api-reference/index.html